ui/components/app/app-header/index.scss

```
@@ -107,4 +92,45 @@
       color: var(--color-primary-inverse);
     }
   }
+
+  &__beta-header {
+    background: var(--brand-colors-orange-orange400);
```

We generally don't want to use `brand-colors` in the app, it could break theming. A more theme appropriate color token would be 
```suggestion
    background: var(--color-warning-default);
```